### PR TITLE
Remove TiFlash http service

### PR DESCRIPTION
**Pull request by Lloyd-Pottiger**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qiancai** [comment]
@Lloyd-Pottiger Please involve a tech reviewer for the tech review. Thanks.

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -40,8 +40,6 @@ aliases: ['/docs-cn/dev/tiflash/tiflash-configuration/','/docs-cn/dev/reference/
 listen_host = "0.0.0.0"
 ## TiFlash TCP 服务的端口
 tcp_port = 9000
```
Comment (qiancai) [inline tiflash/tiflash-configuration.md:42]: 假如用户在之前的 TiDB 版本中配置了这个端口，升级到 7.1 后，是否有什么兼容性变化呢

Comment (Lloyd-Pottiger) [reply tiflash/tiflash-configuration.md:42]: done

**Review by flowbehappy — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**